files/bootstrap.sh

```diff
@@ -142,6 +149,12 @@ Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
 EOF
 fi
 
+if [[ "$ENABLE_DOCKER_BRIDGE" = "true" ]]; then
+    # Enabling live-restore prevents docker from recreating the default bridge network
+    jq '.bridge="docker0" | ."live-restore"=false' /etc/docker/daemon.json | tee  /etc/docker/daemon.json
```

Comment:
Can you reword the comment to agree with the state we are setting here?  E.g. "Disabling..."

Reply:
Sure

Reply:
Nice